tsl/src/nodes/vector_agg/grouping_policy_hash.c

```diff
@@ -225,7 +227,7 @@ add_one_range(GroupingPolicyHash *policy, TupleTableSlot *vector_slot, const int
 	 * Remember which aggregation states have already existed, and which we
 	 * have to initialize. State index zero is invalid.
 	 */
-	const uint32 last_initialized_key_index = policy->last_used_key_index;
+	const uint32 last_initialized_key_index = policy->hashing.last_used_key_index;
```

Comment:
Did some mechanical refactoring to move more hashing-related data into the hashing strategy struct.